src/ActiveRecordInterface.php

```diff
@@ -4,15 +4,15 @@
 
 namespace Yiisoft\ActiveRecord;
 
-use Closure;
 use Throwable;
+use Yiisoft\Arrays\ArrayableInterface;
 use Yiisoft\Db\Exception\Exception;
 use Yiisoft\Db\Exception\InvalidArgumentException;
 use Yiisoft\Db\Exception\InvalidConfigException;
 use Yiisoft\Db\Exception\NotSupportedException;
 use Yiisoft\Db\Exception\StaleObjectException;
 
-interface ActiveRecordInterface
+interface ActiveRecordInterface extends ArrayableInterface
```

Comment:
```suggestion
interface ActiveRecordInterface
```

It is better to move ArrayableInterface to implementation of BaseActiveRecord as you did before

Reply:
ok